e2e/testsuite/diagnostics/diagnostics.go

```diff
@@ -25,11 +23,10 @@ const (
 
 // Collect can be used in `t.Cleanup` and will copy all the of the container logs and relevant files
 // into e2e/<test-suite>/<test-name>.log. These log files will be uploaded to GH upon test failure.
-func Collect(t *testing.T, dc *dockerclient.Client, cfg testconfig.ChainOptions) {
+func Collect(t *testing.T, dc *dockerclient.Client, chainNameA, chainNameB string, debugModeEnabled bool) {
```

Comment:
small nit, I know we already make the assumption that we have 2 chains in 99% of our tests, but might be nice to do

```golang
func Collect(t *testing.T, dc *dockerclient.Client, debugModeEnabled bool, chainNames ...string) {}
```

to allow to support a variable number of chains.